code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -740,8 +740,9 @@
 
 ///Use this to add upgrades to robots. It'll register signals for when the upgrade is moved or deleted, if not single use.
 /mob/living/silicon/robot/proc/add_to_upgrades(obj/item/borg/upgrade/new_upgrade, mob/user)
-	if(new_upgrade in upgrades)
-		return FALSE
+	for(var/obj/item/borg/upgrade/current_upgrade in src.upgrades)
```

Comment:
No need for src, there's nothing in scope named that ?

Reply:
There's no functional difference here. `src.` would only matter if there was another variable named `upgrades` in scope, there isn't one here.

Reply:
functionally now, no, there is no difference.  But not using explicit declarations has the potential to cause issues and makes debugging a living heck, causing the variable to not be accessible through the debugger.  What is the case for omitting them?